examples/40_advanced/example_pandas_train_test.py

```diff
@@ -118,26 +118,30 @@ def get_runhistory_models_performance(automl):
 # ===================================
 
 ensemble_performance_frame = pd.DataFrame(cls.automl_.ensemble_performance_history)
-best_values = pd.Series({'ensemble_optimization_score': -np.inf,
-                         'ensemble_test_score': -np.inf})
+
+best_values = pd.Series({
+    'ensemble_optimization_score': -np.inf,
+    'ensemble_test_score': -np.inf
+})
 for idx in ensemble_performance_frame.index:
-    if (
-        ensemble_performance_frame.loc[idx, 'ensemble_optimization_score']
-        > best_values['ensemble_optimization_score']
-    ):
+    ensemble_score = ensemble_performance_frame.loc[idx, 'ensemble_optimization_score']
+
+    if (ensemble_score > best_values['ensemble_optimization_score']):
         best_values = ensemble_performance_frame.loc[idx]
+
     ensemble_performance_frame.loc[idx] = best_values
 
 individual_performance_frame = get_runhistory_models_performance(cls)
-best_values = pd.Series({'single_best_optimization_score': -np.inf,
-                         'single_best_test_score': -np.inf,
-                         'single_best_train_score': -np.inf})
+best_values = pd.Series({
+    'single_best_optimization_score': -np.inf,
+    'single_best_test_score': -np.inf,
+    'single_best_train_score': -np.inf
+})
 for idx in individual_performance_frame.index:
```

Comment:
Just a note on why I would prefer these if curious. Any editor with tab indicators will have these line up nicely when beginning on a new line as they are lined up as a tab. Contrasted with the hanging version where they are lined up to the previous parameters and not to a tabline may not necessarily be on a tabline.

Really not important but you'll notice me use this a lot more often if possible.

Reply:
I fully agree that this looks a lot nicer.